assets/lokomotive-kubernetes/bootkube/resources/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -239,8 +239,7 @@ spec:
 
           {{ end }}
           backend kube-apiserver
-            option httpchk GET /healthz HTTP/1.1\r\nHost:\ kube-apiserver\r\nAuthorization:\ Bearer\ {{ template "token" . }}
-            server 1 $ADDRESS:{{ template "port" . }} verify none check check-ssl
+            server 1 $ADDRESS:{{ template "port" . }}
```

Comment:
should we squash this with the previous commit? can we explain it a bit more (for example, why don't we have the `check` keyword now?)?

Reply:
The explanation is in the commit:

>As we always have only one backend, it doesn't make sense to perform
those checks.

Simply, HAProxy with checks can remove unhealthy backend from the list of backends, so the requests does not fail. In our case, we only ever have one backend, so we don't need to have health checks, as HAProxy won't return any cached or more specific errors either.

What I think we should have is the liveness probes for both containers, so they are restarted if they stop responding. But I think this is the topic for another issue.

I can actually swap the order of the commits, so we don't add it and then remove it back.

Reply:
ok, a verbose explanation like this comment would be better than the on in the commit :)

Reply:
Added.